refinery/core/models.py

```diff
@@ -1868,9 +1892,14 @@ def _get_output_connection_to_analysis_result_mapping(self):
                     analysis_uuid=self.uuid,
                     file_name=output_connection.filename
                 )[index]
-                output_connections_to_analysis_results.append(
-                    (output_connection, analysis_result)
-                )
+                if output_connection.is_refinery_file:
+                    output_connections_to_analysis_results.append(
+                        (output_connection, analysis_result)
+                    )
+                else:
+                    output_connections_to_analysis_results.append(
+                        (output_connection, None)
+                    )
```

Comment:
So there's always an analysis_result from the database.... but maybe it's not *really* an analysis_result? The if-then could be moved up to where we set it.